.github/workflows/build_paper_old.yml

```diff
@@ -12,21 +10,45 @@ on: [push, pull_request]
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "build"
-  build:
+  # This workflow contains two jobs called "build1" and "build2"
+  build1:
```

Comment:
Would it be worth putting the minted or latex version in the name of the workflow, so that we don't just have 3 similar looking "build" workflows

Reply:
yeah, good point, we can do both i.e., add the docker image release date tag (20240901) and the minted version.

Reply:
I went with just naming the jobs based on the TeXLive version